deploy/helm/kubecf/assets/operations/sequencing.yaml

```diff
@@ -0,0 +1,139 @@
+# Dependencies pointing to dependents.
+# Order here matches order of ops instructions.
+##
+# auctioneer <-- diego-api		(diego only, not eirini)
+# scheduler  <-- api
+# cc-worker  <-- api
+# router     <-- routing-api <-- uaa
+#                tcp-router  <-- uaa
+#                log-cache   <-- uaa
+#                credhub     <-- uaa
+#                diego-cell  <-- uaa	(diego only, not eirini)
+#
+# asactors  <- asdatabase
+# asapi     <- asdatabase
+# asmetrics <- asdatabase
+# asnozzle  <- asdatabase
+##
+
+{{- if not .Values.features.eirini.enabled }}
+
+- type: replace
+  path: /instance_groups/name=auctioneer/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "diego-api" ]'
+
+{{- end }}
+
+- type: replace
+  path: /instance_groups/name=scheduler/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "api" ]'
+- type: replace
+  path: /instance_groups/name=cc-worker/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "api" ]'
+
+- type: replace
+  path: /instance_groups/name=router/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "routing-api" ]'
+
+- type: replace
+  path: /instance_groups/name=routing-api/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+- type: replace
+  path: /instance_groups/name=tcp-router/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+- type: replace
+  path: /instance_groups/name=log-cache/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+- type: replace
+  path: /instance_groups/name=credhub/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+
+{{- if not .Values.features.eirini.enabled }}
+
+- type: replace
+  path: /instance_groups/name=diego-cell/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+
+{{- end }}
+
+#
+# II. Disable upstream pod startup serialization hints.
+#
+# Note: The serialization hints on the various test tasks are
+# irrelevant, and therefore ignored, and not changed.
+#
+
+{{- if not .Values.features.eirini.enabled }}
+
+- type: replace
+  value: false
+  path: /instance_groups/name=auctioneer/update/serial
+
+{{- end }}
+
+- type: replace
+  value: false
+  path: /instance_groups/name=scheduler/update/serial
+- type: replace
+  value: false
+  path: /instance_groups/name=router/update/serial
+- type: replace
+  value: false
+  path: /instance_groups/name=routing-api/update/serial
+
+- type: replace
+  value: false
+  path: /instance_groups/name=api/update/serial
+- type: replace
+  value: false
+  path: /instance_groups/name=diego-api/update/serial
+- type: replace
+  value: false
+  path: /instance_groups/name=log-api/update/serial
+- type: replace
+  value: false
+  path: /instance_groups/name=singleton-blobstore/update/serial
```

Comment:
I _think_ blobstore can be switched off (to use s3)? Not sure on that one.

Reply:
Checking ...

Reply:
Yep. See 
```
features:
  blobstore:
    # Possible values for provider: singleton and s3.
    provider: singleton
    s3:
(...)
```

Hm. There is an error outside of sequencing when I switch to `s3`:
> Error: admission webhook "validate-boshdeployment.quarks.cloudfoundry.org" denied the request: Failed to resolve manifest: Failed to interpolate ops 'ops-configure-bits-service-s3' for manifest 'kubecf' in 'kubecf': Applying ops on manifest obj failed in interpolator: Expected to find exactly one matching array item for path '/instance_groups/name=bits' but found 0

Grepping in the sources shows no file with this. Yet `Binary file bazel-out/k8-fastbuild/bin/deploy/helm/kubecf/cf_deployment.tar matches`. ... An upstream thing, looks like. 
> tar tvf bazel-out/k8-fastbuild/bin/deploy/helm/kubecf/cf_deployment.tar | grep configure-bits
-r-xr-xr-x 0/0             965 1999-12-31 16:00 ./assets/configure-bits-service-s3.yml

Maybe what upstream uses to switch the blobstore providers ? And it is in conflict with our feature ?

Regardless. I will fix the sequencing, and this here needs a separate ticket. And maybe it was just because I disabled internal blobstore without providing proper s3 credentials, etc.

Reply:
Fixed with commit 1233db79137d40221c8bd147de8e8ffbeb67a4f3